alacritty_terminal/src/event_loop.rs

```diff
@@ -327,7 +327,7 @@ where
                         #[cfg(unix)]
                         token if token == self.pty.child_event_token() => {
                             if let Some(tty::ChildEvent::Exited) = self.pty.next_child_event() {
-                                self.terminal.lock().exit();
+                                self.terminal.lock().handle_child_process_exit();
```

Comment:
Why not store `hold` on `self` here and check in here rather than checking in the terminal? Seems a bit complicated to store things on the terminal, if it's only needed here.

Reply:
I did it because I wanted to test it as well, and testing it if it sits on the EventLoop beyond my current skills :) Maybe, since the whole feature boils down to pretty much one `if` statement, tests don't matter that much, but I'm not sure. What do you think?

Reply:
Testing the `exit` method makes sense, but testing a single conditional doesn't hold much value. I'd much prefer a cleaner code style over that personally.

Reply:
Ok, I've moved `hold` to the EventLoop, not the condition is under the `ChildEvent::Exited` match arm and there's no point in having `handle_child_process_exit` on the `Term`, so I've deleted it as well. Much less new code this way :)